spinalcordtoolbox/reports/qc.py

```diff
@@ -750,6 +750,10 @@ def layout(qslice):
             logger.info('open file "%s/index.html"', path_qc)
     except ImportError:
         print("WARNING! Platform undetectable.")
+        # TODO: Remove me!
+        test = None
+        if test is not None:
+            print("Testing coverage!")
```

Comment:
These changes were added just to demonstrate that codecov no longer triggers CI failures for drops in coverage. I'll remove them now.